README.rst

```diff
@@ -24,14 +24,14 @@ PostgreSQL is configured by default to listen to port 5432. Spilo master initial
 
 You'll need to setup Spilo to create a database and roles for your application(s). For example:
 
-    psql -h myfirstspilo.example.com -p 5432 -U admin -d postgres
+    | psql -h myfirstspilo.example.com -p 5432 -U admin -d postgres
```

Comment:
I think this should rather be a `.. code:: bash`?

Reply:
Yes code block make more sense.